pkgs/top-level/all-packages.nix

```diff
@@ -11216,6 +11216,8 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) Cocoa;
   };
 
+  sml = callPackage ../development/libraries/boost-ext/sml {};
```

Comment:
I think sml is a bit generic name. What do you think of using `bext-sml` (like [vcpkg](https://repology.org/project/bext-sml/versions)) or `boost-sml` (like [aur](https://repology.org/project/boost-sml/versions))?

Reply:
I agree. I like `boost-sml` since there is a chance it will be accepted into boost and there would  be no need to change the name.